addons/point_of_sale/static/src/app/services/pos_store.js

```diff
@@ -221,6 +221,9 @@ export class PosStore extends WithLazyGetterTrap {
         if (savedCashier) {
             this.setCashier(savedCashier);
         }
+        else {
```

Comment:
I don't think this will work if you have pos_hr installed. You should only set the cashier like this if pos_hr is installed.